lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -80,6 +70,7 @@ def offending_node?(node)
           num_of_format_args, num_of_expected_fields = count_matches(node)
 
           return false if num_of_format_args == :unknown
+          return false if num_of_expected_fields&.zero? && node.child_nodes.first.dstr_type?
```

Comment:
The test seems to pass on `return false if num_of_expected_fields.zero?` . There seems to be too much or too little either implementation or testing.

Reply:
There seemed to be a lack of testing. Thank you very much.